apps/sms/test/marionette/lib/views/thread_list.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var View = require('./view');
+
+function ThreadList() {
+  View.apply(this, arguments);
+}
+module.exports = ThreadList;
+
+ThreadList.prototype = {
+  __proto__: View.prototype,
+
+  goToFirstThread: function() {
+    this.pageAccessors.ThreadList.firstThread.tap();
+    var Thread = require('./thread');
```

Comment:
Same here, Inbox (former ThreadList, you need to rebase :)) view knows only address (or link) of Conversation view (former ThreadUI), but should not know anything else about it.